apps/python/scart.py

```diff
@@ -1197,31 +1197,12 @@ def usage(exitcode=0):
             if not rs.addStream(
                 stream[2], stream[3], stream[4], stream[5], stream[0], stream[1]
             ):
-                if verbose:
-                    sys.stderr.write(
-                        "error: adding stream: %s %s %s.%s.%s.%s\n"
-                        % (
-                            stream[0],
-                            stream[1],
-                            stream[2],
-                            stream[3],
-                            stream[4],
-                            stream[5],
-                        )
-                    )
+                sys.stderr.write(f"error: adding stream: {stream[0]} {stream[1]} "
```

Comment:
Seems reasonable. Please convert to stderr.write() to print() and use f strings whenever you touch the code.